docs/userGuide/contentAuthoring.md

```diff
@@ -143,10 +143,16 @@ Note: Replace all instances of `-` with `_` in the glyph's name, e.g. <code>{<sp
 
 #### Timestamp
 
-Use <code>{<span></span>{timestamp}}</code> to insert the following snippet that indicates when the page was generated.  
+Use <code>{<span></span>{timestamp}}</code> to insert a UTC time snippet that indicates when the page was generated.  
 
 {{timestamp}}
 
+#### MarkBind reference
+
+Reference or support this project by using <code>{<span></span>{MarkBind}}</code>, which will insert a link back to our website.
+
+{{MarkBind}}

```

Comment:
Now the {{timestamp}} and {{MarkBind}} sample display looks rather out of place. Maybe add to the end of the statement: `This is what it will look like:`, or something similar.